Formula/resolc.rb

```diff
@@ -0,0 +1,33 @@
+class Resolc < Formula
+desc "Revive Solidity Compiler"
+homepage "https://github.com/paritytech/revive"
+url "https://github.com/paritytech/revive/releases/download/v0.1.0-dev.14/resolc-universal-apple-darwin"
+sha256 "7d5b3cd4233e60e8bfaade398062f1b609166dcb251424cd9d402e51126d8b4f"
+license "MIT/Apache-2.0"
+  
+# If someone wants to build from latest source:
+head "https://github.com/paritytech/revive.git", branch: "master"
+depends_on "llvm@18" => :build if build.head?    # only needed when building from source
```

Comment:
This won't work because we need our custom LLVM in order to build resolc. So either remove or add a second Formula for our LLVM build. We also do binary releases of those in our repo.